.forestadmin-schema.json

```diff
@@ -5287,6 +5287,21 @@
           "inverseOf": null,
           "validations": []
         },
+        {
+          "field": "lastLogin",
+          "type": "Date",
+          "defaultValue": null,
+          "enums": null,
+          "integration": null,
+          "isFilterable": true,
+          "isReadOnly": false,
+          "isRequired": false,
+          "isSortable": true,
+          "isVirtual": false,
+          "reference": null,
+          "inverseOf": null,
+          "validations": []
+        },
```

Comment:
I'm curious how you got that modified? Was it done automatically?

This file is a bit new for us, and we're still trying to figure the best practices when modifying it.

Reply:
I did it myself. I was trying to figure out how to make sure my new `lastLogin` column was included everywhere it ought to be and thought that this needed to be modified as well.

Reply:
No Forest API keys were required for that?

Reply:
No - i just edited the file. It didn't fuss or anything 🤔

Reply:
Ok, I get it!

Reply:
I introduced yesterday the first version of a tool to update that file. https://github.com/opencollective/opencollective-api/pull/2694 It's not documented yet and this is why I was interested.